app/assets/stylesheets/announcements.scss

```
@@ -23,6 +23,36 @@
   }
 }
 
+.announcement-border {
+  border: 1px solid $green;
+  border-radius: 1rem;
+  border-top: .45rem solid $green;
+}
+
+.announcement-anchor {
+  color: $white;
+  font-size: .85rem;
+  font-weight: 500;
+  text-decoration: none;
+}
+
+.announcement-text {
+  display: flex;
+  flex-direction: column;
+  font-size: 1rem;
+  font-weight: 500;
+  line-height: 1.5rem;
+  margin-bottom: .5rem;
+}
+
+.announcement-box {
+  background-color: $white;
+}
+
+.an-lg-green {
+  background-color: $card-green;
+}
+
```

```suggestion
.announcement-container {
  background-color: $card-green;
}

```

Keeping it `announcement-container` will be better ig